src/components/structures/auth/ForgotPassword.tsx

```diff
@@ -225,7 +225,7 @@ export default class ForgotPassword extends React.Component<IProps, IState> {
 
     private onInputChanged = (stateKey: string, ev: React.FormEvent<HTMLInputElement>) => {
         this.setState({
-            [stateKey]: ev.currentTarget.value,
+            [stateKey]: (ev.currentTarget.value).trim(),
```

Comment:
we shouldn't be applying this to the password fields though... we'll need an `if` statement.

Reply:
###  Added an If Statement to check if it is the `email` field or not.

```
 private onInputChanged = (stateKey: string, ev: React.FormEvent<HTMLInputElement>) => {
        if( stateKey == "email"){
            this.setState({
                [stateKey]: (ev.currentTarget.value).trim(),
            }as any);
        }else{
            this.setState({
                [stateKey]: ev.currentTarget.value,
            }as any);
        } 
    };
```

Let me know if there are further changes, Thanks !